sktime_dl/deeplearning/cnn/_classifier.py

```diff
@@ -111,6 +108,12 @@ def fit(self, X, y, input_checks=True, **kwargs):
         -------
         self : object
         """
+        if self.random_state is None:
```

Comment:
Ah I see, we convert it here, any reason why we don't use scikit-learn's [`check_random_state`](https://github.com/scikit-learn/scikit-learn/blob/95d4f0841d57e8b5f6b2a570312e9d832e69debc/sklearn/utils/validation.py#L800) for that?

Reply:
self.random_state was previously initialised this way in __init__ so I've done the same here. scikit-learn's check_random_state does a bit more than this line of code, it could be used; the docstring would need to be updated to reflect it as the seed arg would no longer have to be an int. This change could go into a new issue?

Reply:
Opened a separate issue for that #53